packages/generator/src/generators/mutations-generator.ts

```diff
@@ -42,6 +50,13 @@ export class MutationsGenerator extends Generator<MutationsGeneratorOptions> {
       modelNames: this.options.modelNames,
       ModelName: this.options.ModelName,
       ModelNames: this.options.ModelNames,
+      subTemplateValues: this.options.extraArgs?.map((arg: string) => {
+        const [valueName, typeName] = arg.split(":")
+        return {
+          attributeName: singleCamel(valueName),
+          zodTypeName: this.getZodTypeName(typeName),
+        }
+      }),
```

Comment:
- Let's abstract out all these `getTemplateValues` into the base Generator class. Currently all our generators provide the same basic values, so we need to centralize it so they stay consistant. We'll want `subTemplateValues` available in all our generators
  - But we should allow subclasses to add or override the default values.

Reply:
Still todo